kubernetes-client/src/main/java/io/fabric8/kubernetes/client/informers/impl/cache/CacheImpl.java

```diff
@@ -363,12 +358,11 @@ public static String namespaceKeyFunc(String objectNamespace, String objectName)
    * @return the indexed value
    */
   public static List<String> metaNamespaceIndexFunc(Object obj) {
```

Comment:
@manusa Should an exception be thrown if metadata cannot be retrieved like it was before I removed ReflectionUtils?